docs/user/cassandra/commandline_tools.rst

```diff
@@ -0,0 +1,133 @@
+        --key-space mykeyspace \
+        --catalog mycatalog \
+        --name-space mynamespace \
+        --converter example_csv \
+        --spec example_csv \
+        examples/ingest/csv/example.csv
+
+- The ``contact-point`` is the address a Cassandra node. If you're running Cassandra locally with
+  the default configuration, you shouldn't have to change this.
+- The ``key-space`` is the Cassandra key space that you created as part of configuring Cassandra (see above).
+  Edit this argument if needed to match the name of your key space.
+- The ``catalog`` argument specifies the name of a Cassandra table that will be created in your name space
+  by the ``ingest`` command.
+  You can name the catalog table anything you want.
+  This table is a metadata table that contains a list of each GeoMesa data table
+  that you have created. Before you run the ``ingest`` command for the first time, this table does
+  not exist. When you run ``ingest`` for the first time, GeoMesa creates two tables: this "catalog" metadata
+  table, and the table that actually stores your data. If you run ``ingest`` again with a different dataset,
+  GeoMesa will append a row to the "catalog" table, and also create an additional table for the new data.
+- The ``name-space`` parameter can be anything you want. (It is used by GeoTools to provide a name space for your feature types.)
+- The ``converter`` and ``spec`` parameters refer to a specific file located at ``conf/application.conf`` within the
+  ``GEOMESA_CASSANDRA_HOME`` directory. This file contains specifications for "converters" and "sfts" (SimpleFeatureTypes).
+  In this specific example it only contains one of each: a converter and a SimpleFeatureType which are both called
+  "example_csv". The converter specifies how a raw data file should be parsed. For example, the converter specifies
+  that the fourth column of the input file should be converted to a date using a specific date format. The
+  SimpleFeatureType part specifies how the parsed data should be used to create a SimpleFeatureType. For example, this
+  file specifies that the SimpleFeatureType's first attribute should be one called "name". GeoMesa uses these specifications
+  to ingest the data from the external data file into the Cassandra database.
+
+  GeoMesa automatically finds the ``conf/application.conf`` file based on its name and location. You can add additional
+  converters and SFT specifications to it as needed for ingesting other datasets. In addition, you can also
+  specify converters and SFT specifications by adding directories to the
+  ``conf/sfts`` directory. For more details see :ref:`installing_sft_and_converter_definitions`.
+- The last argument is the location of the data file that we want to ingest.
+
+If needed, edit the parameter arguments in the bash script. Then, ``cd`` into the ``GEOMESA_CASSANDRA_HOME``
+directory, and run the script by typing::
+
+  source examples/ingest/csv/ingest_csv.sh
```

Comment:
I have not run this myself, but all it's doing is executing ``geomesa-cassandra``, so I think that just running it would work.